invenio_records_resources/services/uow.py

```diff
@@ -187,14 +187,15 @@ def on_commit(self, uow):
 class IndexRefreshOp(Operation):
     """Search index refresh operation."""
 
-    def __init__(self, index, **kwargs):
+    def __init__(self, indexer, index, **kwargs):
```

Comment:
The indexer obj has access to the `record_cls` which allows us to already access the index already. See [here](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/services/records/service.py#L51). I'm 99% sure we can remove the `index` parameter.